frontend/appflowy_flutter/assets/translations/hi.json

```diff
@@ -0,0 +1,441 @@
+{
+  "appName": "ऐप फ्लोवी",
+  "defaultUsername": "में",
+  "welcomeText": "स्वागत है @:appName",
+  "githubStarText": "गिटहब पैर स्टार दे",
+  "subscribeNewsletterText": "हमारे न्यूज़ की जानकारी पाए",
+  "letsGoButtonText": "शुरू करे",
+  "title": "टाइटल",
+  "signUp": {
+    "buttonText": "साईन करे",
+    "title": "साईन करे @:appName",
+    "getStartedText": "शुरू करे",
+    "emptyPasswordError": "पासवर्ड खाली नहीं रख सकते",
+    "repeatPasswordEmptyError": "रिपीट पासवर्ड खाली नहीं रख सकते",
+    "unmatchedPasswordError": "रिपीट पासवर्ड और पहला पासवर्ड सामान नहीं है",
+    "alreadyHaveAnAccount": "क्या आपके पास पहले से ही खाता है ?",
+    "emailHint": "ईमेल",
+    "passwordHint": "पासवर्ड",
+    "repeatPasswordHint": "रिपीट पासवर्ड"
+  },
+  "signIn": {
+    "loginTitle": "लॉगिन करे @:appName",
+    "loginButtonText": "लॉगिन",
+    "loginAsGuestButtonText": "शुरू करे",
+    "buttonText": "साईन करे",
+    "forgotPassword": "क्या आप पासवर्ड भूल गए ?",
+    "emailHint": "ईमेल",
+    "passwordHint": "पासवर्ड",
+    "dontHaveAnAccount": "क्या आपके पास पहले से ही खाता है ?",
+    "repeatPasswordEmptyError": "रिपीट पासवर्ड खाली नहीं रख सकते",
+    "unmatchedPasswordError": "रिपीट पासवर्ड और पहला पासवर्ड सामान नहीं है"
+  },
+  "workspace": {
+    "create": "काम करने की जगह बनाये",
```

Comment:
`workspace` should either remain as 'वर्कस्पेस' or `कार्यस्थान` because here it is a noun, we could go with `काम करने की जगह` but I think `कार्यस्थान` is better, what you think @AmanNegi

Reply:
Yeah rightly said @MayurSMahajan, `काम करने की जगह` doesn't specify anything useful. I guess it would be better to keep `वर्कस्पेस`.